packages/api/client.ts

```diff
@@ -68,22 +68,49 @@ export const getOperators = async (dbClient: Kysely<Database>, input: OperatorQu
 export type StopsQueryInput = {
     atcoCodes?: string[];
     naptanCodes?: string[];
+    commonName?: string;
+    adminAreaCode?: string;
     page?: number;
 };
 
 export const getStops = async (dbClient: Kysely<Database>, input: StopsQueryInput) => {
     logger.info("Starting getStops...");
 
-    const STOPS_PAGE_SIZE = 50;
+    const STOPS_PAGE_SIZE = 1000;
 
-    if (input.atcoCodes || input.naptanCodes) {
+    if (input.adminAreaCode) {
+        logger.info("Filtering by admin area code...");
+        const stopsByAdminAreaCode = await dbClient
+            .selectFrom("stops")
+            .selectAll()
```

Comment:
I've asked Steve what data we should actually need and he said the following:
```
  "id"
  "atcoCode"
  "naptanCode"
  "commonName"
  "street"
  "indicator"
  "bearing"
  "nptgLocalityCode"
  "localityName"
  "parentLocalityName"
  "longitude"
  "latitude"
  "stopType"
  "busStopType"
  "timingStatus"
  "administrativeAreaCode"
  "status"
```
so we should be able to trim down the response size a bit

Reply:
done